### PR TITLE
chore: update packages and regen audit resolve

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: src/domain/consents/ID.ts [hunk 1]
```diff
@@ -77,29 +77,31 @@ export async function updateConsentCredential (
...
       credentialType: 'FIDO',
       status: CredentialStatusEnum.VERIFIED,
-      challenge: {
-        payload: consent.credentialChallenge as string,
-        signature: signature
-      },
-      payload: publicKey
+      payload: {
+        id: consent.credentialId!,
+        rawId: 'TODO: figure out what goes here',
```
Comment (eoln) [inline src/domain/consents/ID.ts:98]: `rawId` is a byte array made from string  `id`

File: src/model/scope/scope.ts [hunk 2]
```diff
@@ -40,6 +40,8 @@ import { NotFoundError } from '../errors'
 
 /*
 * Interface for Scope resource type
+* TODO: rename this so it doesn't conflict with the Thirdparty API's
+*       Scope interface. Maybe something like AuthDBScope or ScopeDBRowObject
```
Comment (eoln) [inline src/model/scope/scope.ts:44]: `ModelScope` ?

**Review by eoln — APPROVED** [review]

File: src/domain/consents/ID.ts (hunk 1 above)
Comment (kleyow) [reply src/domain/consents/ID.ts:98]: Ah. Thank you. I'll get that in my pass that I'm working on today.

File: src/model/scope/scope.ts (hunk 2 above)
Comment (kleyow) [reply src/model/scope/scope.ts:44]: That works.

**Pull request merged**